src/components/AccessibilityGuide/StickyElements.mdx

```diff
@@ -0,0 +1,77 @@
+import { Meta, Source } from "@storybook/blocks";
+
+import Link from "../Link/Link";
+
+<Meta title="Accessibility Guide/Sticky Elements" />
+
+# Sticky Elements
+
+## Table of Contents
+
+- {<Link href="#general-information" target="_self">General Information</Link>}
+- {<Link href="#implementation" target="_self">Implementation</Link>}
+- {<Link href="#resources" target="_self">Resources</Link>}
+
+## General Information
+
+**Edwin/Marty: Why do we use sticky headers or sticky elements in general?**
```

Comment:
Want to include a couple lines here about the design rationale for sticky elements

Reply:
Here are a few resources for sticky elements on sites:
- https://www.smashingmagazine.com/2023/05/sticky-menus-ux-guidelines/ 
- https://technology.blog.gov.uk/2018/05/21/sticky-elements-functionality-and-accessibility-testing/
- https://design.gs.com/patterns/sticky-elements

It looks like the general pattern is to use sticky headers/menus/elements when it is useful for users to understand where they are on a site, keep them within a certain process flow, or to convey information that we think will be useful.

But for NYPL, I'd argue that we should try to stay away from sticky components where possible and only use them when we need to convey critical or specific information. I think the DC sticky header is a feature that existed and we want parity from the old app to the new app for consistency and feel.

@bigfishdesign13 and I would need a bit more time for this "General Information" context.